packages/hardhat-plugin-ethers/src/ethers-ignition-helper.ts

```diff
@@ -145,6 +145,12 @@ export class EthersIgnitionHelper {
       defaultSender,
       strategy,
       strategyConfig: resolvedStrategyConfig,
+      maxFeePerGasLimit:
+        this._hre.config.networks[this._hre.network.name]?.ignition
+          .maxFeePerGasLimit,
+      maxPriorityFeePerGas:
+        this._hre.config.networks[this._hre.network.name]?.ignition
```

Comment:
We should consider whether the deploy on the ethers/viem helpers should expose an override for both `maxFeePerGasLimit` and `maxPrioirityFeePerGas`.
  
Is it just unlikely to come up in test scenarios, given they are against Hardhat node?

Reply:
I think it was simply an oversight that they were not already exposing these overrides. I think both helpers should expose any options available for normal deployment, specifically because of script based deployments.

Reply:
or are you suggesting adding additional overrides that can be passed in via the `deploy` function itself? I'm less inclined to provide this pattern unless it's requested by users.

Reply:
Lets leave this for now. We can add it later if we get requests.